static/app/utils/gettingStartedDocs/node.ts

```diff
@@ -124,6 +136,66 @@ export function getDefaulServerlessImports({
   return imports;
 }
 
+export function getNodeRunCommandSnippet(): string {
+  return `# If you are using CommonJS (CJS)
+node --require ./instrument.js app.js
+
+# If you are using ECMAScript Modules (ESM) - Note that this is only available from Node 18.19.0 onwards.
+node --import ./instrument.mjs app.mjs`;
+}
+
+export function getImportInstrumentSnippet(): string {
+  return `// IMPORTANT: Make sure to import \`instrument.js'\` at the top of your file.
+  // If you're using ECMAScript Modules (ESM), use \`import './instrument.js';\`
+  require('./instrument.js');`;
```

Comment:
l-m: I'm afraid this conflicts a bit with users' interpretation of when they're using ESM to write vs to run. I have a couple of ideas but feel free to come up with your own or table this for now and merge anyway:

```suggestion
  // If you're using ECMAScript Modules (ESM) syntax, use \`import './instrument.js';\`:
  require('./instrument.js');`;
```


or

```suggestion
  // If you're `import` syntax, replace this with \`import './instrument.js';\`:
  require('./instrument.js');`;
```

Reply:
Good suggestion! I can make it more clear that this is about syntax.